src/dune_pkg/package_universe.ml

```diff
@@ -55,7 +55,7 @@ let concrete_dependencies_of_local_package t local_package_name ~with_test =
          (Solver_env.to_env t.solver_env)
          t.version_by_package_name
   with
-  | Ok s -> s
+  | Ok { regular; _ } -> regular
```

Comment:
Nitpick, but maybe better use `post=_` to discard it in case new fields get added as we want the compiler to complain.